pkg/rehearse/jobs_test.go

```diff
@@ -181,11 +183,6 @@ func TestMakeRehearsalPresubmitNegative(t *testing.T) {
 		description string
 		crippleFunc func(*prowconfig.Presubmit)
 	}{{
-		description: "job with multiple containers",
```

Comment:
Why?

Reply:
this validation is already happening when we load the prowconfigs.